pkgs/development/libraries/glibc/common.nix

```
@@ -63,8 +63,8 @@ stdenv.mkDerivation ({
          and using git or something would complicate bootstrapping.
          Fortunately it's not too big.
           $ git checkout origin/release/2.32/master; git describe
-          glibc-2.32-39-g44b3959329
-          $ git show --reverse glibc-2.32.. | gzip -n -9 --rsyncable - > 2.32-master.patch.gz
+          glibc-2.32-40-g778b8d3786
+          $ git show --minimal --reverse glibc-2.32.. | gzip -n -9 --rsyncable - > 2.32-master.patch.gz
```

```suggestion
          $ git show --minimal --reverse glibc-2.32.. | gzip -n -9 --rsyncable - > 2.32-master.patch.gz
          
          To compare the archive contents zdiff can be used.
          $ zdiff -u $file $file2
```

maybe a good information for reviewers.